discord/message.py

```diff
@@ -1991,6 +1993,9 @@ def to_message_reference_dict(self) -> MessageReferencePayload:
 
         return data
 
+    def to_dict(self) -> MessagePayload:
```

Comment:
Can you add a docstring documenting this ? Doesn't need to be fancy, just what's needed.